.github/scripts/requirements.txt

```diff
@@ -0,0 +1,22 @@
+autopep8==1.6.0
```

Comment:
We should only be installing the minimum required packages to do this test. Am I crazy or are most of the packages not used?